modules/editor/message/PrimaryContentEditor.tsx

```diff
@@ -25,28 +29,43 @@ export function PrimaryContentEditor(props: PrimaryContentEditorProps) {
         error={form.field("content").error}
         {...form.field("content").inputProps}
       />
-      <RowContainer>
-        <InputField
-          id={`_${message.id}_username`}
-          label="Username"
-          maxLength={80}
-          error={form.field("username").error}
-          {...form.field("username").inputProps}
-        />
-        <InputField
-          id={`_${message.id}_avatar`}
-          label="Avatar URL"
-          error={form.field("avatar").error}
-          {...form.field("avatar").inputProps}
-        />
-      </RowContainer>
-      <FileInputField
-        id={`_${message.id}_files`}
-        label="Files"
-        maxSize={8 * 1024 ** 2}
-        value={message.files}
-        onChange={files => message.set("files", files)}
-      />
+      <Section hasError={false} name="Extras">
```

Comment:
hasError is an optional prop
```suggestion
      <Section name="Extras">
```

Reply:
this is what it looks like without it
![image](https://user-images.githubusercontent.com/16510197/108555762-f86fb300-7306-11eb-80e3-f0bfe53b8f8d.png)

Reply:
Oh, that shouldn't have happened. Set a default in `common/layout/Section.tsx` by doing `hasError = false`